test/new_relic/agent/monitors/cross_app_monitor_test.rb

```diff
@@ -157,8 +157,14 @@ def test_writes_metric
     def test_doesnt_write_metric_if_id_blank
       when_request_runs(for_id(''))
 
-      assert_metrics_recorded_exclusive(['transaction', 'Supportability/API/drop_buffered_data',
-        'OtherTransactionTotalTime', 'OtherTransactionTotalTime/transaction'])
+      assert_metrics_recorded_exclusive([
+        'transaction',
+        'Supportability/API/drop_buffered_data',
+        'OtherTransactionTotalTime',
+        'OtherTransactionTotalTime/transaction',
+        'Supportability/API/record_metric',
+        'Supportability/Deprecated/cross_application_tracer.enabled configuration'
```

Comment:
I'm not sure if we are allowed to have a space or period in a metric like this. I don't think I've seen any metrics that have either of those characters. When we get back we should look into that and what the limitations are on the server. Since it's coming from the call to the deprecator, that's where we'd need to update what we're calling the metric.

Reply:
@tannalynn - how would we go about looking into the limitations on the server for the metrics?

Reply:
The agent specs seem to prefer underscores only as separators for the last metric section. Nice catch!